h/static/scripts/directive/markdown.coffee

```diff
@@ -328,6 +328,23 @@ module.exports = ['$filter', '$sanitize', '$sce', '$timeout', ($filter, $sanitiz
             $sanitize textToCheck.substring(startMath, endMath)
       return textToCheck
 
+    scope.checkLength = (text) ->
```

Comment:
I don't understand what this section is for. Isn't the excerpt directive supposed to be responsible for all this?